apps/minifront/src/components/v2/header/prax-popover.tsx

```diff
@@ -0,0 +1,55 @@
+import { useMemo } from 'react';
+import { Link2Off } from 'lucide-react';
+import { Popover } from '@penumbra-zone/ui/Popover';
+import { Button } from '@penumbra-zone/ui/Button';
+import { Text } from '@penumbra-zone/ui/Text';
+import { penumbra, usePraxManifest } from '../../../prax.ts';
+import praxLogo from './prax.svg?url';
+
+const getIconComponent = (blob?: Blob, name?: string) => {
+  const Icon = () => {
+    return <img src={blob ? URL.createObjectURL(blob) : praxLogo} alt={name} className='size-4' />;
```

Comment:
issue (blocking): if no icon is present, a potentially misleading default should not be displayed.